ironic-source-list.j2

```diff
@@ -0,0 +1,18 @@
+crudini
+gunicorn
+{% if env.IRONIC_SOURCE_HASH %}
+ironic @ git+https://opendev.org/openstack/ironic@{{ env.IRONIC_SOURCE_HASH }}
+{% elif env.IRONIC_SOURCE_GERRIT %}
+ironic @ git+https://opendev.org/openstack/ironic@{{ env.IRONIC_SOURCE_GERRIT }}
```

Comment:
Looks identical to the previous case, maybe use the same option?

Reply:
yes, I will use the same option
it will be one option for each custom source